composer.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "answear/poczta-polska-bundle",
+  "description": "Poczta Polska integration for Symfony.",
+  "type": "symfony-bundle",
+  "license": "MIT",
+  "require": {
+    "php": ">=7.3",
+    "ext-json": "*",
+    "ext-zip": "*",
+    "ext-simplexml": "*",
+    "marc-mabe/php-enum": "^3.0",
+    "webmozart/assert": "^1.3",
+    "symfony/validator": "^4.1",
+    "symfony/http-kernel": "^4.1",
+    "php-http/guzzle6-adapter": "^1.1",
+    "symfony/serializer": "^4.4",
+    "symfony/property-access": "^4.1"
```

Comment:
Why all these requirements? I see no guzzle or validator usages in #2

Reply:
:+1: Left only http-kernel from those above as it contains Bundle class